met-web/src/services/userService/index.ts

```diff
@@ -107,8 +110,18 @@ const hasRole = (role: string) => KeycloakData.hasResourceRole(role);
 
 const hasAdminRole = () => KeycloakData.hasResourceRole(AppConfig.keycloak.adminRole);
 
+const updateUser = async () => {
+    try {
+        await http.PutRequest(Endpoints.User.CREATE_UPDATE);
+        // TODO: update store with current user info.
+    } catch (e: unknown) {
+        console.error(e);
```

Comment:
should we consider logging the user out if this put request failed? Are there any big concerns here if this put request failed and the user was left to log in?